robfallows/tunguska-reactive-aggregate
# Improve id handling

## Description

See discussion in #4 
Closes #3.

## Testing

To test this PR:

1. Create a `packages` folder in your project's root, if you don't already have one.
1. `cd packages`
1. `git clone git@github.com:robfallows/tunguska-reactive-aggregate.git`
1. `cd tunguska-reactive-aggregate`
1. `git checkout improve-id-handling`
1. `cd ../..`
1. `meteor`

You should see that Meteor reports an update to `tunguska:reactive-aggregate`.

You now need to run any tests that were failing before the package update.

## Feedback

1. Please ensure you provide feedback to this PR as to whether it fixes your issue or not.
1. Please ensure you provide feedback to the PR if it introduces any new issues.

## Release

Following a successful PR review, I will release the updated package to atmosphere. You may then delete the cloned copy of the package from your project.
